packages/quantic/force-app/main/default/lwc/quanticUserActionEvent/quanticUserActionEvent.js

```diff
@@ -0,0 +1,169 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * @typedef Document
+ * @property {string} title
+ * @property {string} [clickUri]
+ * @property {string} [uriHash]
+ * @property {string} [contentIdKey]
+ * @property {string} [contentIdValue]
+ */
+
+/**
+ * @typedef UserAction
+ * @property {'search' | 'click' | 'view' | 'custom' | 'case-creation' | 'active-case-creation'} actionType
```

Comment:
Looks like we're using "ticket" rather than "case" in the display and in the headless props so I recommend we stay consistent here as well.
 
```suggestion
 * @property {'search' | 'click' | 'view' | 'custom' | 'ticket-creation' | 'active-ticket-creation'} actionType
```